src/util/template-builder.js

```diff
@@ -0,0 +1,21 @@
+
+const templateTypeToTemplateMap = {
+  pr: require('~/src/util/templates/pr-template')
+}
+
+function renderToStringAsync (template, data) {
+  return new Promise((resolve, reject) => {
+    template.renderToString(data, (err, html) => {
+      err ? reject(err) : resolve(html)
+    })
+  })
+}
+
+exports.build = async function (typeName, data) {
+  let template
+  if (!(template = templateTypeToTemplateMap[typeName])) {
```

Comment:
There doesn't seem to be much gain here by using assignment in the conditional here. This could easily be a `const` above and a simple falsey check on this line. Easier to read imo, and no change in the number of lines. Just my opinion, though.